pkg/objstore/s3/s3.go

```diff
@@ -38,6 +38,17 @@ const (
 // DirDelim is the delimiter used to model a directory structure in an object store bucket.
 const DirDelim = "/"
 
+// s3Config stores the configuration for gcs bucket
+type s3Config struct {
+	Bucket        string `yaml:"bucket"`
+	Endpoint      string `yaml:"endpoint"`
+	AccessKey     string `yaml:"access-key"`
+	SecretKey     string `yaml:"secret-key"`
```

Comment:
And now is that real question.. should it be available here or just through envvar (: I remember people saying it is not good idea to specify them via config file or flags

Reply:
OK, I will rollback it to just use envvar (: